networking/hardware_networks/configuring-sriov-device.adoc

```diff
@@ -5,11 +5,12 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can configure a Single Root I/O Virtualization (SR-IOV) device in your cluster.
+You can configure a Single Root I/O Virtualization (SR-IOV) device in your cluster. For the detailed object specification, refer to xref:../../networking/hardware_networks/configuring-sriov-device.adoc#nw-sriov-networknodepolicy-object_configuring-sriov-device[SR-IOV network node configuration object].
 
-include::modules/nw-sriov-device-discovery.adoc[leveloffset=+1]
 include::modules/nw-sriov-configuring-device.adoc[leveloffset=+1]
-include::modules/nw-sriov-nic-partitioning.adoc[leveloffset=+1]
+include::modules/nw-sriov-networknodepolicy-object.adoc[leveloffset=+1]
```

Comment:
How this paragraph is not included in virt-sriov doc? I assume with current change, virt-doc uses nw-sriov-configuring-device.adoc for node policy explanation.

Reply:
@zshi-redhat, I added a conditional so that the original content remains the same for OpenShift Virtualization. I think we need to work with @aburdenthehand to incorporate the IB content.